app/controllers/general.php

```diff
@@ -837,6 +837,9 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
             case 'Utopia\Database\Exception\Dependency':
                 $error = new AppwriteException(AppwriteException::INDEX_DEPENDENCY, null, previous: $error);
                 break;
+            case 'Utopia\Database\Exception\Index':
+                $error = new AppwriteException(AppwriteException::INDEX_INVALID, $error->getMessage(), previous: $error);
+                break;
```

Comment:
We want to move away from catching expected exceptions in the general handler, let's add this only where needed in the databases controller/worker

Reply:
100% removed 👍